05-bootstrap-prep.md

```diff
@@ -79,7 +79,8 @@ We'll be bootstrapping this cluster with the Flux GitOps agent as installed as a
    Now commit changes to repository.
 
    ```bash
-   git commit -a -m "Update image source to use my ACR instance instead of a public container registry."
+   git add ./cluster-manifests/cluster-baseline-settings/kured.yaml
+   git commit -m "Update image source to use my ACR instance instead of a public container registry."
```

Comment:
Why make this two commands?  Their shouldn't have been any other changes that aren't already git ignored.  Seems like a maint problem to have to always remember to maintain this changed file list.  We've avoided that elsewhere before.

Reply:
I had changed SKUs and availability zones (you may know why...) in cluster-stamp.bicep that I had not wanted to commit. But I do not insist on that, if you think it may be better to leave it as it was.

Reply:
Understood the reasoning in your case.  Yeah, let's just leave it at one command to simplify the steps (and maint).  If someone is "off the path" for other reasons, they can figure out how to just commit what they needed to commit.  We'll stay true on the happy path :)

Reply:
Thanks for reverting this.